compiler/test/stdlib/option.test.gr

```diff
@@ -124,3 +124,7 @@ assert c == 4
 let mut d = '🌾'
 assert Option.peek((x) => fail 'Should not be called', None) == None
 assert d == '🌾'
+
+# Option.toResult
+assert Option.toResult(Some(1), "Nope") == Ok(1)
```

Comment:
This didn't get switched to data-last, but I'll get this merged and follow up with a PR that changes it and adds the Option methods to mirror this PRs awesome additions (like `or` and `and`)